Tests/js/views/ez-view.html

```diff
@@ -24,7 +29,7 @@
         filter: loaderFilter,
         modules: {
             "ez-view": {
-                requires: ['view', 'base-pluginhost', 'ez-pluginregistry'],
+                requires: ['view', 'base-pluginhost', 'ez-pluginregistry', 'event-tap'],
```

Comment:
that's not the right place to add `event-tap`, the test depends on it not the implementation so it should be added at the end of ez-view-tests.js